Sources/RxAlamofire.swift

```diff
@@ -784,6 +790,25 @@ extension Reactive where Base: DataRequest {
         }
     }
 
+    public func responseJSON() -> Observable<DataResponse<Any>> {
+        return Observable.create { observer in
+            let request = self.base
+
+            request.responseJSON { response in
+                if let error = response.result.error {
+                    observer.onError(error)
+                } else {
+                    observer.onNext(response)
+                    observer.onCompleted()
+                }
+            }
+
+            return Disposables.create {
```

Comment:
Uhm yeah sure I'll add that! I usually prefer doing the closure for readability but for a one liner that's nicer for sure. Thanks !